### DISCUSSION TITLE
Display Progress

### DISCUSSION THREAD

**Discussion by royrajan95** [kalanakt/All-Url-Uploader#89] [General]
hi,
in Documet option, Unable add caption after link. bot shows ERROR... Incorrect Link

 in Video option
no progress bar or details while downloading..
but while uploading it shows

{} 

📤 Uploading Please Wait 
 [◾️◾️◽️◽️◽️◽️◽️◽️◽️◽️◽️◽️◽️◽️◽️◽️◽️◽️◽️◽️] 
P: 11.71%
229.5 MB of 1.91 GB

Speed: 4.56 MB/s

ETA: 7m, 10s
Can you fix caption error & add downloading progress too??

**Comment by kalanakt** [comment]
w.o. it
